gdal/ogr/ogrsf_frmts/flatgeobuf/ogrflatgeobufdataset.cpp

```diff
@@ -0,0 +1,302 @@
+#include "ogr_flatgeobuf.h"
+
+#include "header_generated.h"
+
+static int OGRFlatGeobufDriverIdentify(GDALOpenInfo* poOpenInfo){
+    if( STARTS_WITH_CI(poOpenInfo->pszFilename, "FGB:") )
+        return TRUE;
+
+    if( STARTS_WITH(poOpenInfo->pszFilename, "/vsicurl") )
+    {
+        if( CPLGetValueType(CPLGetFilename(poOpenInfo->pszFilename)) ==
```

Comment:
is this logic needed ? It seems it comes from the PBF driver

Reply:
Unsure, will have to dive into it more. I have problems to fully understand the logic involving mapping files/directories to datasets.

Reply:
Removed for now..